README.md

```diff
@@ -15,6 +15,12 @@ Run the development server:
 ```bash
 pnpm dev
 ```
+Run a snyk scan localy
+If snyk is configure do a local snyk test
```

Comment:
How do we configure snyk to run locally 🙈

Reply:
Setup instructions in line number 20